ARCHITECTURE.md

```diff
@@ -0,0 +1,100 @@
+## Architecture 
+More insight information about the way `material-tools` works.
+
+#### Quick Links
+- [How does it work](#how-does-it-work)
+- [How to run Angular in NodeJS](#how-to-run-angular-in-nodejs)
+- [Why are some files manually compiled](#why-are-some-type-of-files-compiled-manually)
+- [Under-the-Hood: Understanding the Virtual Context](#under-the-hood-understanding-the-virtual-context)
+- [Under-the-Hood: How to generate a static theme](#under-the-hood-how-to-generate-a-static-theme)
+
+### How does it work
+Material Tools allows developers to build a custom version of [Angular Material 1.x](http://www.github.com/angular/material).
+
+As a developer you could just specifiy the given [components](https://github.com/angular/material/tree/master/src/components) 
```

Comment:
I personally don't like that wording here. 

What do you think about `Creating a custom Material Build`?